projects/ngrx.io/content/guide/migration/v13.md

```diff
@@ -17,7 +17,7 @@ Version 13 has the minimum version requirements:
 - Angular version 13.x
 - Angular CLI version 13.x
 - TypeScript version 4.4.x
-- RxJS version 7.4.x
+- RxJS version 6.5.3 or 7.4.x
```

Comment:
To avoid (new) confusion that the exact `6.5.3` version is required, I suggest the following:

```suggestion
- RxJS version 6.5.x, 6.6.x, or 7.4.x
```

Reply:
Yes I also thought about this, but I found it a bit weird since it is already specified **minimum** version requirements. Agree that your suggestion reads better and will apply it. 

Perhaps one should adapt the npm semantics and use for example `^6.5.3 || ^7.4.0`, then drop the "minimum"